tests/acl/test_acl.py

```diff
@@ -119,13 +119,22 @@ def setup(duthost, testbed):
     duthost.command('rm -rf {}'.format(DUT_TMP_DIR))
 
 
-@pytest.fixture(scope='module', params=['ingress', 'egress'])
-def stage(request):
+@pytest.fixture(scope="module", params=["ingress", "egress"])
```

Comment:
Just curios, is quotation mark change for consistency or for any weird behavior?

Reply:
There's a mix of singles and doubles in this file at the moment. I'll go through and update it all to be consistent, but I want to open another PR for that since I think there are some other ACL test fixes coming in soon.